textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/join/JoinDistanceHelper.java

```diff
@@ -0,0 +1,139 @@
+                JoinTestConstants.BOOK_SCHEMA, LuceneAnalyzerConstants.standardAnalyzerString());     
+        relationManager.createTable(BOOK_TABLE_INNER, "../index/jointest/book_inner/", 
+                JoinTestConstants.BOOK_SCHEMA, LuceneAnalyzerConstants.standardAnalyzerString());         
+    }
+    
+    public static void insertToOuter(ITuple... tuples) throws StorageException {
+        RelationManager relationManager = RelationManager.getRelationManager();
+        for (int i = 0; i < tuples.length; i++) {
+            relationManager.insertTuple(BOOK_TABLE_OUTER, tuples[i]);
+        }
+    }
+    
+    public static void insertToOuter(List<ITuple> tuples) throws StorageException {
+        RelationManager relationManager = RelationManager.getRelationManager();
+        for (ITuple tuple: tuples) {
+            relationManager.insertTuple(BOOK_TABLE_OUTER, tuple);
+        }
+    } 
+    
+    public static void insertToInner(ITuple... tuples) throws StorageException {
+        RelationManager relationManager = RelationManager.getRelationManager();
+        for (int i = 0; i < tuples.length; i++) {
+            relationManager.insertTuple(BOOK_TABLE_INNER, tuples[i]);
+        }
+    }
+    
+    public static void insertToInner(List<ITuple> tuples) throws StorageException {
+        RelationManager relationManager = RelationManager.getRelationManager();
+        for (ITuple tuple: tuples) {
+            relationManager.insertTuple(BOOK_TABLE_INNER, tuple);
+        }
+    } 
+    
+    public static void clearTestTables() throws TextDBException {
+        RelationManager relationManager = RelationManager.getRelationManager();
+        relationManager.deleteTuples(BOOK_TABLE_OUTER, new MatchAllDocsQuery());
+        relationManager.deleteTuples(BOOK_TABLE_INNER, new MatchAllDocsQuery());      
+    }
+    
+    public static void deleteTestTables() throws TextDBException {
+        RelationManager relationManager = RelationManager.getRelationManager();
+        relationManager.deleteTable(BOOK_TABLE_OUTER);
+        relationManager.deleteTable(BOOK_TABLE_INNER);
+    }
+    
+    public static KeywordMatcherSourceOperator getKeywordSource(String tableName, String query, 
+            KeywordMatchingType matchingType) throws TextDBException {
+        DataStore tableDataStore = RelationManager.getRelationManager().getTableDataStore(tableName);
+        KeywordPredicate keywordPredicate = new KeywordPredicate(query, 
+                Arrays.asList(JoinTestConstants.AUTHOR, JoinTestConstants.TITLE, JoinTestConstants.REVIEW),
+                LuceneAnalyzerConstants.getStandardAnalyzer(), matchingType);
+        KeywordMatcherSourceOperator keywordSource = new KeywordMatcherSourceOperator(keywordPredicate, tableDataStore);
+        return keywordSource;
+    }
+    
+    public static List<ITuple> getJoinDistanceResults(IOperator outerOp, IOperator innerOp,
```

Comment:
This helper functions wraps the logic for creating a Join Operator and returning the result tuples in a list.